### PR TITLE
WIP: Add support for launching Windows nodes in AWS clusters from cluster-bot

### PR DESCRIPTION
**Pull request by jrvaldes**
Add support for bringing up IPI clusters on AWS with
Windows nodes. This will be used from cluster-bot by
https://github.com/openshift/ci-chat-bot/pull/251

### CODE REVIEW

File: ci-operator/step-registry/windows/conf/operator/windows-conf-operator-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+ref:
```
Comment (aravindhp) [inline ci-operator/step-registry/windows/conf/operator/windows-conf-operator-ref.yaml:1]: Can't you somehow leverage the [operatorhub](https://github.com/openshift/release/tree/master/ci-operator/step-registry/operatorhub) steps?

**Comment by jrvaldes** [comment]
Closed to leverage the new ci-chat-bot feature [workflow-launch](https://github.com/openshift/release/blob/master/core-services/ci-chat-bot/workflows-config.yaml) in https://github.com/openshift/release/pull/28387.

**Pull request closed without merging**